### PR TITLE
Convert remaining SupportPower uses of seconds to ticks

### PR DESCRIPTION
**Pull request by reaperrr**
Logn overdue, in my opinion.

### CODE REVIEW

File: OpenRA.Mods.Cnc/Traits/SupportPowers/GpsPower.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ namespace OpenRA.Mods.Cnc.Traits
 	[Desc("Requires `GpsWatcher` on the player actor.")]
 	class GpsPowerInfo : SupportPowerInfo
 	{
+		[Desc("Delay of map reveal in ticks.")]
```
Comment (pchote) [inline OpenRA.Mods.Cnc/Traits/SupportPowers/GpsPower.cs]: "Delay in ticks between launching and revealing the map"?

File: OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ class IonCannonPowerInfo : SupportPowerInfo, IRulesetLoaded
 		[Desc("Actor to spawn when the attack starts")]
 		public readonly string CameraActor = null;
 
-		[Desc("Amount of time to keep the camera alive")]
+		[Desc("Amount of ticks to keep the camera alive")]
```
Comment (pchote) [inline OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs]: "Number of ticks to keep the camera alive"

File: OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs [hunk 3]
```diff
@@ -15,8 +15,8 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public abstract class SupportPowerInfo : PausableConditionalTraitInfo
 	{
-		[Desc("Measured in seconds.")]
-		public readonly int ChargeTime = 0;
+		[Desc("Measured in ticks.")]
+		public readonly int ChargeDelay = 0;
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: Can we please call this `Interval` instead? Our naming convention is to use *Delay for one-off events, and Interval for things that repeat.

**Comment by reaperrr** [comment]
Updated.

**Comment by MustaphaTR** [comment]
Needs rebase.

**Comment by reaperrr** [comment]
Rebased.

**Review by abcdefg30 — DISMISSED** [review]

**Comment by reaperrr** [comment]
Minor update (see last commit) and rebased.

**Review by GraionDilach — APPROVED** [review]
:+1:

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/44ca09a04849408435d73a3838f502495c7fab6e

**Pull request merged**